sql/moz-fx-data-shared-prod/telemetry_derived/adm_engagements_daily_v1/metadata.yaml

```
@@ -0,0 +1,17 @@
+---
+friendly_name: adMarketplace Daily Engagements
+description: >
+  Click Volume, partitioned by day with the following facets:
+    - submission_date
+    - phase
+    - country
+    - placement
+    - partner
+    - version
+    - normalized_channel
+owners:
+  - tbrooks@mozilla.com
```

@teonbrooks, is this OK? I want to end your responsibility to this query, but I also don't want to plagiarize your work.

```suggestion
  - rharter@mozilla.com
```